cpp/unreal_plugins/Vehicle/Source/Vehicle/Vehicle.cpp

```diff
@@ -5,19 +5,25 @@
 #include "Vehicle/Vehicle.h"
 
 #include <Modules/ModuleManager.h> // IMPLEMENT_GAME_MODULE, IMPLEMENT_MODULE
+#include <ChaosVehicleMovementComponent.h>
 
 #include "SpCore/AssertModuleLoaded.h"
 #include "SpCore/Log.h"
+#include "SpCore/UnrealClassRegistrar.h"
 
 void Vehicle::StartupModule()
 {
     SP_ASSERT_MODULE_LOADED("SpCore");
     SP_LOG_CURRENT_FUNCTION();
+
+    UnrealClassRegistrar::registerComponentClass<UChaosVehicleMovementComponent>("UChaosVehicleMovementComponent");
```

Comment:
Let's move these to `SpComponents.cpp`. We're already registering some types there anyway, and I want to get rid of the `Vehicle` plugin soon, since it isn't doing anything useful. (The entire plugin could be replaced with a single component in `SpComponents` that inherits from `UChaosVehicleMovementComponent`.)